### PR TITLE
 Fix - 1773 Fixing Scrolling incase toggle button is off

### LINKED ISSUE #1773: Fixing Scrolling incase toggle button is off
**I'm submitting a ...**
- [x] bug report

In case toggle is in off state scroller should check for conditions which is currently throwing error .
Lower one error. 

**Current behaviour:**

screenshot from 2018-10-09 20-05-54

**Expected behaviour:**

First it needs to be checked whether toggler is off or not.

**Steps to reproduce:**
Switch toggle button off and try to scroll the page . View console 

Will fix it ??
Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1773 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Incase of toggle button is off , the earlier code was generating style error because the card was not found. Implementing a check on card.

ezgif com-video-to-gif 14

@ParthS007 @abishekvashok  @gabru-md  Please review it.

### CODE REVIEW

File: api/config/config.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 DEBUG = True
 
 # Environment for working, can be LOCAL/PROD
-ENV = 'PROD'
+ENV = 'LOCAL'
```
Comment (rajvaibhavdubey) [inline api/config/config.py]: @sk9331657 this ENV variable should be set to 'PROD' only.

**Comment by sk9331657** [comment]
@ParthS007 @gabru-md  @yashhere @yashLadha  Either please suggest me whether to use the external library which needs to be added because there is no as such event binders for scrolling in Ember.js and or this can be merged with new changes.

**Comment by sk9331657** [comment]
@gabru-mg Please suggest what should be done for this pr.

File: frontend/app/components/create-badge.js [hunk 2]
```diff
@@ -4,19 +4,23 @@ const EmberObject  = Ember;
...
-        EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.position = 'fixed';
-        EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '570px';
-      } else {
-        EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px';
-        EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.position = 'inherit';
-
+    EmberObject.$(window).bind('resize scroll', function(e) {
+      if (EmberObject.$(document)[0].getElementsByClassName('six wide column preview').length != 0) {
+        if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && this.scrollY < 1278) {
+          EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.position = 'fixed';
```
Comment (yashLadha) [inline frontend/app/components/create-badge.js]: Create a variable and then used at different places. Modularisation :sweat_smile:

**Comment by sk9331657** [comment]
@gabru-md What should be done??

**Comment by sk9331657** [comment]
@yashLadha  @gabru-md  @ParthS007

**Comment by sk9331657** [comment]
@ParthS007  Please suggest What should be done for this.

**Comment by gabru-md** [comment]
@ParthS007 reviews?

**Comment by sk9331657** [comment]
@ParthS007  @gabru-md  @yashLadha  Please review.

**Comment by sk9331657** [comment]
@yashLadha  @ParthS007 Please review and merge this.

**Comment by sk9331657** [comment]
@ParthS007

**Pull request merged**